src/components/graphs/myDashboardBar/myDashboardBar.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useMemo } from 'react';
+import { VictoryAxis, VictoryBar, VictoryChart, VictoryStack, VictoryTheme, VictoryTooltip } from 'victory';
+import maxBy from 'lodash/maxBy';
+import { useTheme } from '../../../providers';
+import { MyDashboardTooltip } from '../tooltip/MyDashboardTooltip';
+
+export interface MyDashboardBarData {
+  isoWeek: number;
+  duration: number;
+}
+
+export interface MyDashboardBarProps {
+  data: MyDashboardBarData[];
+  width?: number;
+  weekText?: string;
+  hoursText?: string;
+  shortHoursText?: string;
+  tooltipWidth?: number;
+  tooltipHeight?: number;
+}
+
+export const MyDashboardBar = ({
+  data,
+  width = 1400,
+  weekText = 'Week',
+  hoursText = 'hours',
+  shortHoursText = 'h',
+  tooltipWidth = 150,
+  tooltipHeight = 100,
+}: MyDashboardBarProps) => {
+  const {
+    theme: {
+      current: {
+        customColors: { light6, purple2 },
+        colors: { contentInverseTertiary },
+        sizing: { scale400 },
+      },
+    },
+  } = useTheme();
+
+  const barData = useMemo(() => {
+    return data.map((item) => ({
+      ...item,
+      duration: item.duration / 3600,
+    }));
+  }, [data]);
+
+  const maxValue = useMemo(() => {
+    const maxValueDataItem = maxBy(barData, (dataItem) => dataItem.duration);
```

Comment:
I'm not sure that we need to add lodash to project just for one function
If we need to find max value by some attribute in array of objects, I'd suggest to use something like it
```
 Math.max.apply(Math, barData.map((dataItem) => dataItem.duration))
 ```

Reply:
make sense now, but it still will be just for one function further